test/test_helper.rb

```diff
@@ -13,8 +13,8 @@
 require "debug"
 require "mocha/minitest"
 
-sorbet_paths = Gem.loaded_specs["sorbet-runtime"].full_require_paths.freeze
-DEBUGGER__::CONFIG[:skip_path] = Array(DEBUGGER__::CONFIG[:skip_path]) + sorbet_paths
+SORBET_PATHS = T.let(Gem.loaded_specs["sorbet-runtime"].full_require_paths.freeze, T::Array[String])
```

Comment:
Changed to a constant that we can reference it from `BacktraceWithoutSorbetFilter`.